presto-docs/src/main/sphinx/release/teradata-additions.rst

```diff
@@ -0,0 +1,48 @@
+==================
+Teradata Additions
+==================
+
+The following are additions that Teradata has added on top Facebook's 0.148 release of Presto
+
+Prepared Statements
+-------------------
+Add support for Prepared statements and parameters via sql syntax.
+
+    * PREPARE
+    * DEALLOCATE PREPARE
+    * EXECUTE
+    * DESCRIBE INPUT
+    * DESCRIBE OUTPUT
+
+Data Types
+----------
+Add FLOAT support to the Hive connector
+Add Char support to Hive connector
+Additional Varchar(x) function implementations
+Additional Decimal functions implementations
+
+Documentation
+-------------
+Additional Kerberos
+Grant/Revoke
+Presto-Admin
+Presto YARN Integration
+Presto Ambari Integration
+TINYINT, SMALLINT, INTEGER
```

Comment:
I could remove it since it's misleading. I mean we added the documentation addition.